lyse/communication.py

```diff
@@ -0,0 +1,125 @@
+#####################################################################
+#                                                                   #
+# /main.py                                                          #
```

Comment:
Incorrect filename (there could be other instances of this - please check all the new files)